### PR TITLE
Splash Screen

### PR DESCRIPTION
**Pull request by DanielFirpo**
Thought it might be cool to add a splash screen, especially since the title on page was removed

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,6 +8,9 @@
         <meta name="description" content="Watch youtube with a twitch stream simultaneously." />
         <link name='favicon' id='apple-icon' rel="apple-touch-icon" href="%PUBLIC_URL%/iwd.png" />
         <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+        <!-- Not sure if this is necessary, but rly want to make sure this image is loaded before splashscreen renders, so preload -->
+        <link rel="preload" href="../src/splashlogo.png" as="image">
```
Comment (brianjp93) [inline public/index.html:13]: I think the correct way to access this here is to use `%PUBLIC_URL%` like above, right?

File: src/components/caster/caster.jsx [hunk 2]
```diff
@@ -35,6 +36,7 @@ export function Caster(props) {
     <div>
       {casterQuery.data && (
         <>
+        <SplashScreen></SplashScreen>
```
Comment (brianjp93) [inline src/components/caster/caster.jsx:39]: just do `<SplashScreen />` if there's no content inside.

File: public/index.html (hunk 1 above)
Comment (brianjp93) [reply public/index.html:13]: Writing it this way will try to access https://iwdlive.com/src/splashlogo.png

File: src/components/general/SplashScreen.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
+import React, { useEffect } from 'react'
+import SplashLogo from "../../splashlogo.png"
```
Comment (brianjp93) [inline src/components/general/SplashScreen.jsx:2]: I think this will have a similar problem
https://create-react-app.dev/docs/using-the-public-folder/

**Review by brianjp93 — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (brianjp93) [reply public/index.html:13]: I think you'll have to move the splashlogo into the public folder as well

Comment (DanielFirpo) [reply public/index.html:13]: I'll just remove that from the html, I don't think it's even necessary I was just worried that the image might not be ready in time for the splash screen.

File: src/components/caster/caster.jsx (hunk 2 above)
Comment (DanielFirpo) [reply src/components/caster/caster.jsx]: done

**Pull request merged**